cloud-formation/README.md

```diff
@@ -1,12 +1,37 @@
-# CloudFormation scripts
+# Cloud Formation
 
-These scripts require the [AWS CloudFormation Command Line Tools][1].
+The [template](./dev-template.json) outlines the AWS resources needed to develop on the Grid.
 
- [1]: http://aws.amazon.com/developertools/2555753788650372
+## Creating Stack
 
-## Troubleshooting
+Annoyingly, in order to create a lambda function within a Cloud Formation template whose code is in S3, the code artifact has to pre-exist within a S3 bucket.
```

Comment:
Note that this shouldn't be an issue once we start deploying lambda's via riff-raff.